mw-webapp/src/component/confirm/Confirm.tsx

```diff
@@ -2,10 +2,26 @@ import {ReactElement, useRef, useState} from "react";
 import {Close as DialogClose} from "@radix-ui/react-dialog";
 import {Button, ButtonType} from "src/component/button/Button";
 import {HorizontalContainer} from "src/component/horizontalContainer/HorizontalContainer";
-import {Modal} from "src/component/modal/Modal";
+import {Cy, Modal} from "src/component/modal/Modal";
 import {KeySymbols} from "src/utils/KeySymbols";
 import styles from "src/component/confirm/Confirm.module.scss";
 
+/**
+ * Data attributes for cypress testing
+ */
+interface cyDataConfirm extends Cy {
```

Comment:
Start name of interfaces with big letter

Reply:
Done